plugins/lookup/grafana_dashboard.py

```diff
@@ -116,7 +114,15 @@ def grafana_switch_organisation(self, headers):
     def grafana_headers(self):
         headers = {'content-type': 'application/json; charset=utf8'}
         if self.grafana_api_key:
-            headers['Authorization'] = "Bearer %s==" % self.grafana_api_key
+            api_key = self.grafana_api_key
+            if len(api_key) % 4 == 2:
```

Comment:
Not sure to understand this :sweat_smile:  ?

Reply:
The '=' characters at the end of base64 encoded content are to pad it out to a multiple of 4. In order for the interface to remain backwards compatible, it needs to continue adding '==' to the API key when the length is off by 2.